database/mysql/0019_discovery_monitoring.sql

```diff
@@ -0,0 +1,229 @@
+/*
+ *
+ * Copyright (C) 2017-2018 Eaton
+ *
+ * This software is confidential and licensed under Eaton Proprietary License
+ * (EPL or EULA).
+ *
+ * This software is not authorized to be used, duplicated or disclosed to
+ * anyone without the prior written permission of Eaton.
+ * Limitations, restrictions and exclusions of the Eaton applicable standard
+ * terms and conditions, such as its EPL and EULA, apply.
+ *
+ */
+
+/*
+ * @file    0019_discovery_monitoring_v2.sql
```

Comment:
Is the new NUT configurator expected to be merged right now? I was under the impression it had to be modified to switch from an asset-centric to an endpoint-centric approach.

Reply:
Thanks for the nudge, maybe the file leaked from some other work or experiment, not sure it was destined for deb10 specific work indeed. At least does not seem to fit to the subject of https://github.com/42ity/fty-core/commit/0e3e835c79b697bd850973c35b7287aa509b538c (the `/var/run => /run` changes) so might be just something `git add`ed in the workspace and forgotten :\

Reply:
Rebased to drop it from the PR.